scss/_navbar.scss

```diff
@@ -153,6 +153,14 @@
 // Custom navbar navigation (doesn't require `.nav`, but does make use of `.nav-link`).
 
 .navbar-nav {
+  // scss-docs-start navbar-nav-css-vars
+  --#{$prefix}nav-link-padding-x: var(--#{$prefix}navbar-nav-link-padding-x); // Boosted mod
+  --#{$prefix}nav-link-padding-y: #{$nav-link-padding-y};
```

Comment:
This is a modification in BS right?

Reply:
Yes but since we override the values : `--#{$prefix}nav-link-padding-y: var(--#{$prefix}navbar-nav-link-padding-y); // Boosted mod` would be better here IMO, but can maybe be tackled in another PR